.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!-- Dear friend, we, the rest of the NativeScript community really appreciate your feedback! While we are doing all we can to take care of every issue, sometimes we get overwhelmed. Because of that, we will consider issues that are not constructive or problems that cannot be reproduced "dead". Additionally, we will treat feature requests or bug reports with unanswered questions regarding the behavior/reproduction for more than 20 days "dead". All "dead" issues will get closed. -->
+
+## Please, provide the details below:
+
+### Did you verify this is a real problem by searching [Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?
+
+### Tell us about the problem
+<!--Please, ensure your title is less than 63 characters long and starts with a capital
+letter. -->
+
+### Please provide the following version numbers that your issue occurs with:
+<!--
+- CLI: (run `tns --version` to fetch it)
+- Cross-platform modules: (check the 'version' attribute in the
+`node_modules/tns-core-modules/package.json` file in your project)
+- Runtime(s): (look for the `"tns-android"` property in the
+`package.json` file of your project)
+- Plugin(s): (look for the version number in the `package.json` file of your
+project)
+-->
+
+### Did the error happen while the app was being constructed? (buildtime error)
+<!--
+Please provide us with the log by running the command:
+```
+tns <command> --log trace
+```
+Attach the content of the log to the Issue. You can use [pastebin](http://pastebin.com/).
+-->
+
+### Did the error happen while the app was executing? (runtime error)
+<!--
+Please provide us with the log by following these steps:
+* Add `__enableVerboseLogging()` in the `app.js` file.
+* Run command: `adb logcat`.
+* Attach the content of the log to the Issue. You can use [pastebin](http://pastebin.com/).
+-->
+
+### Please tell us how to recreate the issue in as much detail as possible.
+<!--
+Please provide us with the necessary information to reproduce your problem. That includes but is not limited to presenting us with steps to reproduce the problem, or a small repo example.
+-->
+
```

Comment:
I think this part better stay in the comments. But if we decide otherwise, I will add such a section.